Code/Editor/EditorFramework/Assets/Implementation/AssetWatcher.cpp

```diff
@@ -258,7 +258,7 @@ void ezDirectoryUpdateTask::Execute()
   {
     CURATOR_PROFILE("FindReferencedFiles");
     // Find all currently known files that are under the given folder.
-    // TODO: is m_InverseDependency / m_InverseReferences covered by this?
+    // TODO: is m_InverseTransformDeps / m_InverseThumbnailDeps covered by this?
```

Comment:
is this comment still relevant ?

Reply:
Nope, I don't think they are relevant to the current code anymore.